plugins/gatsby-remark-mdx-v2-fenced-code-blocks/index.js

```diff
@@ -0,0 +1,25 @@
+const { createCompiler } = require('@mdx-js/mdx');
+const visit = require('unist-util-visit');
+
+const INVALID_CODE_BLOCK = /`{3,}\n/;
```

Comment:
It seems like we're relying a lot on regex for all these transform operations we've been doing (which is great, this is not a complaint).

Perhaps we should consider putting all these regex statements in one location (or maybe even wrap them in a helper class with some common functionality). Either way, that's out of scope for this PR.

Reply:
> putting all these regex statements in one location 

This is definitely an idea, though I'd rather put them in a spot due to _function_, not just because they are regexes. These are just a tool like anything else (function, variable, etc.) so I don't necessarily think we should treat them as unique. For example, we don't have a file just called "functions". That being said, if there are regexes that are copied across several locations, there is definitely an opportunity to consolidate those.

As for the helper function, it didn't seem like wrapping this in a function provided much value, hence why I tried to make the variable name descriptive. This didn't seem to really give me much:

```
const isInvalidCodeBlock = (string) => INVALID_CODE_BLOCK.test(string)
```

I'd rather just use `INVALID_CODE_BLOCK.test(string)` directly.